### PR TITLE
s/buck/buck2/ in users.md

### PR DESCRIPTION
**Pull request by stepancheg**
Change link, title and company name.

### CODE REVIEW

**Review by laurentlb — APPROVED** [review]

**Comment by laurentlb** [comment]
We have to wait for @brandjon and @tetromino to merge the PR (I no longer have write access)

**Review by tetromino — APPROVED** [review]

**Comment by tetromino** [comment]
Thanks!

**Pull request merged**
